### DISCUSSION TITLE
are devtools going to be fixed at any point: firing actions twice

### DISCUSSION THREAD

**Discussion by noxjonas** [ngrx/platform#3332] [Q&A]
I see issues opened in 2017 (https://github.com/ngrx/platform/issues/1190) and 2018 (https://github.com/ngrx/platform/issues/1054) were both closed without an actual solution. It is 2022 and the bug is still here.

this is directly related to importing/using the devtools module, otherwise actions work fine. Will this ever be fixed?

I also noticed that upon ng install it adds the 'logOnly' option for production, but this does not prevent actions from firing twice...

**Comment by timdeschryver** [comment]
As pointed out in both tickets, a reproduction is important to find the cause.
A lot of the comments in the related tickets resulted that the problem wasn't caused by devtools.
